src/plugins/format.js

```diff
@@ -62,12 +63,27 @@
 				}
 			}
 
-			if (!this.commands.format) {
-				this.commands.format = {
-					exec: formatCmd,
-					txtExec: formatCmd,
-					tooltip: 'Format Paragraph'
-				};
+			sceditor.command.set('format', {
+				exec: formatCmd,
+				txtExec: formatCmd,
+				tooltip: 'Format Paragraph'
```

Comment:
It would have been:
`tooltip: instance._('Format Paragraph')`
but I had failed to notice that SCE already calls it automatically at the time the buttons are generated.

Reply:
ok. I've copied that as is from defaultCommands.js.
So besides that everything is ok?

Reply:
Yes. I didn't have the ability to test, unfortunately. My life has been hellish to find any time frame for more than 5 mins at random times.... Which I need to test these stuff before merging.